test/framework/applications/guestbooktest.go

```diff
@@ -135,6 +133,12 @@ func (t *GuestBookTest) DeployGuestBookApp(ctx context.Context) {
 		"cluster": map[string]interface{}{
 			"enabled": false,
 		},
+		"podSecurityPolicy": map[string]interface{}{
```

Comment:
Isn't podSecurityPolicy introduced as of v9.5.0 of the redis chart - https://github.com/helm/charts/pull/18425/files? 
It looks like we are using v7.0.0 of the chart - https://github.com/gardener/gardener/blob/dba80d637da3da853e4cf3128fee9a989344cfd3/test/framework/framework_test.go#L61.

Reply:
you are referring to just the unit test for the test framework. The actual guestbook test uses 10.2.1 https://github.com/gardener/gardener/blob/master/test/framework/applications/guestbooktest.go#L49

Reply:
You are right, my bad. :(